gsod.md

```diff
@@ -23,7 +23,7 @@ Here are some possible projects:
 [SciML](https://sciml.ai/) is the scientific machine learning organization. However, 
 its documentation is spread amongst many different fairly large packages:
 
-- [DifferentialEquations.jl](https://docs.sciml.ai/latest/)
+- [DifferentialEquations.jl](https://docs.sciml.ai/)
```

Comment:
let's remove this whole project: the unified documentation should launch tonight 😅 fingers crossed.